.iex.exs

```diff
@@ -0,0 +1 @@
+alias GlassFactoryApi.{ApiClient, Member}
```

Comment:
As @peagha once said in a review of a Playfair pull request you can also add here `import_file_if_available("~/iex.exs")` to import the user global aliases if they exist. 😄